apis/core-api/src/api/routes/v1/users/ControllerAdapter.ts

```diff
@@ -155,6 +155,24 @@ class ControllerAdapter {
 
     return ctx.ok(response);
   }
+
+  static async revokeToken(ctx: Context): Promise<void> {
+    /** VALIDATIONS, PARAMETERS */
+    const authToken = (ctx.headers['authorization'] || '').replace('Bearer ', '');
```

Comment:
Why this check is here, havn't we a middleware which will set it into header?

Reply:
In middleware, I'm just validating and/or updating auth token (if expires soon or expired), but I don't modifying them.